### PR TITLE
Update unity.md

### PR DESCRIPTION
**Pull request by vegaro**
Fixed observer mode instructions since they were outdated

### CODE REVIEW

File: docs_source/🚀 Getting Started/installation/unity.md [hunk 1]
```diff
@@ -86,47 +88,57 @@ If you have enabled Minify in Unity, make sure to add these custom rules to your
...
+**side by side with Unity IAP 4.8.0**
 
-**side by side with Unity IAP 3.3.0 < 4.4.0**
+This version is only compatible with version 4.8.0 and above of Unity IAP which are the ones that include BillingClient 5.
 
-Download `Purchases-UnityIAP.unityPackage` and install as normal. Skip the rest of the instructions in this page. 
+To install download `Purchases.unityPackage` and install as normal. Then follow the instructions indicated in the "Troubleshooting "duplicated class" errors"
 
----
+## Purchases Unity 4.2.0+
```
Comment (tonidero) [inline docs_source/🚀 Getting Started/installation/unity.md:108]: I wonder if we should move these instructions further below, considering these are installation steps for older SDK versions... Considering that the `Troubleshooting "duplicated class" errors` section now it's much more important, I think that should be before these.

File: docs_source/🚀 Getting Started/installation/unity.md [hunk 2]
```diff
@@ -86,47 +88,57 @@ If you have enabled Minify in Unity, make sure to add these custom rules to your
...
-[block:callout]
-{
-  "type": "danger",
-  "body": "Make sure to use version [3.2.0](https://github.com/RevenueCat/purchases-unity/releases/tag/3.2.0) of our plugin. Due to incompatibilities with the version of BillingClient used by Unity IAP, the latest versions of the plugin won't work alongside Unity IAP until Unity IAP gets updated to BillingClient 4.x.x.",
-  "title": "Unity IAP doesn't work with version 3.3.0<4.0.0 of the purchases plugin"
-}
-[/block]
 ---
 
 ## Troubleshooting "duplicated class" errors
```
Comment (tonidero) [inline docs_source/🚀 Getting Started/installation/unity.md:124]: I wonder if we should just make this part of the main instructions...

**Review by tonidero — APPROVED** [review]
A couple of comments but LGTM!

File: docs_source/🚀 Getting Started/installation/unity.md (hunk 1 above)
Comment (vegaro) [reply docs_source/🚀 Getting Started/installation/unity.md:108]: Good point, I wil do that

**Pull request merged**